app/src/main/res/layout/fragment_time_sheet.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.fragment_time_sheet` appears to be unused
 Unused resources make applications larger and slow down builds.